ISL.ReIdentification.Core/Services/Foundations/UserAccesses/UserAccessService.Validations.cs

```diff
@@ -94,6 +94,12 @@ private async ValueTask ValidateUserAccessOnModifyAsync(UserAccess userAccess)
                 (Rule: await IsNotRecentAsync(userAccess.UpdatedDate), Parameter: nameof(UserAccess.UpdatedDate)));
         }
 
+        private async ValueTask ValidateUserAccessOnRemoveById(Guid userAccessId)
+        {
+            Validate(
+                (Rule: await IsInvalidAsync(userAccessId), Parameter: nameof(UserAccess.Id)));
+        }
```

Comment:
Could simplify it to this since it is just a one liner.

```suggestion
        private async ValueTask ValidateUserAccessOnRemoveById(Guid userAccessId) =>
            Validate((Rule: await IsInvalidAsync(userAccessId), Parameter: nameof(UserAccess.Id)));
```